minimesos/build.gradle

```diff
@@ -96,8 +96,19 @@ task installMinimesosScript(type: Copy) {
     into "/usr/local/bin"
 }
 
+task sourcesJar(type: Jar, dependsOn: classes) {
+    classifier = 'sources'
+    from sourceSets.main.allSource
+}
+
+task javadocJar(type: Jar, dependsOn: javadoc) {
+    classifier = 'javadoc'
+    from javadoc.destinationDir
+}
 
 artifacts {
+    archives sourcesJar
+    archives javadocJar
```

Comment:
Guess you could argue wether is good practice to do these two steps here. It will inevitably affect the day to day build for most developers.